examples/js/ShaderTerrain.js

```diff
@@ -84,8 +84,6 @@ THREE.ShaderTerrain = {
 			"varying vec3 vNormal;",
 			"varying vec2 vUv;",
 
-			"uniform vec3 ambientLightColor;",
```

Comment:
Not in sea3d, It was harnesses the PR to fix the example webgl_terrain_dynamic.html

`THREE.WebGLProgram: shader error:  0 gl.VALIDATE_STATUS false gl.getProgramInfoLog invalid shaders  ERROR: 0:158: 'ambientLightColor' : redefinition`